### DISCUSSION TITLE
switch_server 이벤트는 언제 발생하는 것인가요?

### DISCUSSION THREAD

**Discussion by secneuqes** [storycraft/node-kakao#535] [Q&A]
현재 봇 가동중이며, 코드에 switch_server 코드 또한 포함 시켰습니다. 그러나 현재까지의 콘솔기록을 살펴본 결과 스위치 서버가 전혀 작동하지 않는듯 보입니다. 혹시 이 이벤트가 어떤 상황에서 발생하는지 알려주실 수 있나요?

**Comment by wntown** [comment]
서버가 꺼지거나 소켓연결이 끊겼을때도 이벤트 발생 하는거 같습니다
switch_server 이벤트 실행 하시고
다른 디바이스에서 로그인 해보시면 확인 되실겁니다

**Reply by secneuqes** [reply]
저도 그래서 switch_server 코드를 넣고 다른 디바이스에서 로그인을 force로 해보았지만 여전히 발생되지 않는것을 확인하였습니다.
```
client.on('switch_server', () => {
    login().then(() => {
        console.log('Server switched!');
    });
});
```
위와 같은 코드를 넣고 실행 했음에도 작동하지 않는 이유가 있을까요?
